gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -148,6 +135,72 @@ def kerchunkify(self, file_path: str, scan_indices: int = 0, local_file_path: Op
 
         return scanned_zarr_json
 
+    def local_kerchunk(self, file_path: str, scan_indices: int = 0):
```

Comment:
The function definition is missing the annotation for the return value, and the annotation for scan_indices I think should be `int | slice` or `int | Tuple[int]`. There's a bit of info [here](https://stackoverflow.com/questions/76125621/how-to-annotate-slice-type-in-python) on why slice is hard to annotate. Also just a reminder we should be consistent about leaving the annotations out of the docstring.

Reply:
Putting slices in the docstrings caused all hell to break loose previously.

Reply:
ok